tests/x11/remote_desktop/xrdp_client.pm

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Remote Login: SLED access Windows over RDP
+# Maintainer: GraceWang <gwang@suse.com>
+# Tags: tc#1610392
+
+use strict;
+use base 'x11test';
+use testapi;
+use lockapi;
+use mmapi;
+use mm_tests;
+
+sub run {
+    my $self = shift;
+
+    # Setup static NETWORK
+    $self->configure_static_ip_nm('10.0.2.17/24');
+
+    mutex_lock 'win_server_ready';
+
+    # Start Remmina and login the remote server
+    x11_start_program('remmina', target_match => 'remmina-launched');
+    type_string '10.0.2.18';
+    send_key 'ret';
+    wait_still_screen 3;
+    send_key 'alt-o';
+
+    # input account name
+    wait_still_screen 3;
+
+    # We can not use the variable $realname here
+    # Since the windows username limit is 20 characters
+    # The username in windows is different from the $realname
+    type_string "Bernhard M. Wiedeman";
+    send_key "tab";
+    type_password;
+    wait_still_screen 3;
+    send_key "ret";
+
+    wait_still_screen 3;
+    check_screen "connection-failed";
+    if (match_has_tag 'connection-failed') {
+        record_soft_failure 'bsc#1117402 - Remmina is not able to connect to the windows server';
+        send_key "alt-f4";
+    }
+    else {
+        assert_screen "windows-desktop-on-remmina";
+        assert_and_click "close-remote-connection";
+    }
```

Comment:
This part could use `assert_screen` with multi-tags?

Reply:
Below are the code using assert_screen with multi-tags:
`assert_screen [qw(windows-desktop-on-remmina close-remote-connection)];
assert_and_click "close-remote-connection" if match_has_tag('close-remote-connection');`

I think the original way is more simple.